src/components/Card/__tests__/Card.test.tsx

```diff
@@ -210,16 +210,22 @@ describe('Card', () => {
     test('ignore onClick if type!="container"', async () => {
         const user = userEvent.setup();
         const handleOnClick = jest.fn();
+        const outerHandler = jest.fn();
         render(
-            <Card type="container" onClick={handleOnClick} qa={qaId}>
-                {cardText}
-            </Card>,
+            <div onClick={outerHandler} onKeyDown={outerHandler}>
+                <Card type="container" onClick={handleOnClick} qa={qaId}>
+                    {cardText}
+                </Card>
+            </div>,
         );
 
         const card = screen.getByText(cardText);
 
         await user.click(card);
-        expect(handleOnClick).toHaveBeenCalledTimes(0);
+        await user.keyboard('[Space]');
+        await user.keyboard('[Enter]');
+        expect(handleOnClick).not.toHaveBeenCalled();
+        expect(outerHandler).toHaveBeenCalled();
```

Comment:
What does it test? Is active element inside `Сard` after click?

Reply:
1. onCLick handler is not called for Card with type container
2. onKeyDown is not fired on div element, because active element is not inside it.

> Test that we didnt prevent outside handlers

then check should be this `expect(outerHandler).toHaveBeenCalledTimes(3);`, right?

Reply:
Yes, we can count invocations too

Reply:
I've changed test to case that failed for us